packages/react/src/components/Authenticator/ConfirmSignUp/ConfirmSignUp.tsx

```diff
@@ -38,6 +39,17 @@ export function ConfirmSignUp() {
     placeholder: translate('Enter your code'),
   };
 
+  const deliveryText =
+    codeDeliveryDetails.DeliveryMedium === 'EMAIL' ? 'Emailed' : 'Texted';
+
+  const confirmSignUpHeading = translate(`We ${deliveryText} You`);
+
+  const subtitleText =
+    translate(`Your code is on the way. To log in, enter the code we
+            ${deliveryText.toLowerCase()} to
+            ${codeDeliveryDetails.Destination}. It may take a minute to
+            arrive.`);
+
```

Comment:
Because of the template literal, `translate` would be difficult to override because of the extra spacing & `\n`s.

I know it's repetitive and not super-DRY, but would this work?

```suggestion
  const { DeliveryMedium, Destination } = codeDeliveryDetails;
  const confirmSignUpHeading = DeliveryMedium === 'EMAIL'
    ? translate('We emailed you')
    : translate('We texted you')
  const subtitleText = DeliveryMedium === 'EMAIL'
    ?	`Your code is on the way. To log in, enter the code we emailed to ${Destination}. It may take a minute to arrive.`)
    : `Your code is on the way. To log in,  enter the code we texted to ${Destination}. It may take a minute to arrive.`
```

It's less templating, but I think the benefit of find/replace helps for identifying messaging. (Not that it's final)

This would also have problems with the parameterization of `Destination` & translations, so maybe we can't really use `translate` here.

Reply:
I agree, might be difficult to translate here. So I'll make these changes.

Reply:
I went ahead and added another level to the ternary if the code delivery is not SMS or EMAIL, or it doesn't exist. I could have changed this to an if/else if/else or a switch. However, I think it's still readable. 

```
  const confirmSignUpHeading =
    DeliveryMedium === 'EMAIL'
      ? translate('We Emailed You')
      : DeliveryMedium === 'SMS'
      ? translate('We Texted You')
      : translate('We Sent A Code');
  const subtitleText =
    DeliveryMedium === 'EMAIL'
      ? `Your code is on the way. To log in, enter the code we emailed to ${Destination}. It may take a minute to arrive.`
      : DeliveryMedium === 'SMS'
      ? `Your code is on the way. To log in, enter the code we texted to ${Destination}. It may take a minute to arrive.`
      : translate(
          `Your code is on the way. To log in, enter the code we sent you. It may take a minute to arrive.`
        );
```

Reply:
I'm good with it, I like ternaries 😈